students/kristianjf/lesson02/print_grid.py

```diff
@@ -0,0 +1,58 @@
+'''
+Part 1: Write a function that draws a grid like the following:
++ - - - - + - - - - +
+|         |         |
+|         |         |
+|         |         |
+|         |         |
++ - - - - + - - - - +
+|         |         |
+|         |         |
+|         |         |
+|         |         |
++ - - - - + - - - - +
+'''
+a = '+ '
+b = '- ' * 4
+c = '| '
+d = '  ' * 4
+x = str('{}{}{}{}{}\n').format(a, b, a, b, a)
+y = str('{}{}{}{}{}\n').format(c, d, c, d, c)
+print('Part 1: Write a function that draws a grid\n'+x+y*4+x+y*4+x)
```

Comment:
Whoa I really like that you used .format() and "\n" and completely eliminated the need for looping. Very clever!